src/pages/billing/Project.js

```diff
@@ -369,11 +365,11 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
             }
           })
         ])])]),
-        CostCard({ title: 'Total spend', amount: (!!totalCost ? totalCost : '$__.__') })
+        CostCard({ title: 'Total spend', amount: (totalCost !== null ? totalCost : '$__.__') })
```

Comment:
Consider

```suggestion
        CostCard({ title: 'Total spend', amount: (totalCost ?? '$__.__') })
```

for concision and also because (I'd think) we'd want to display the default when `totalCost` is `undefined` as well.

Reply:
The default value is null, it should never be undefined.